docs/en/features/census_configuration.md

```diff
@@ -1,155 +1,145 @@
 # Configure connection to the Census
 
-The objective of this service is to be able to configure the connection with the Town Hall Census through the Administration panel without having to modify the application code.
+The objective of this service is to be able to configure the connection with a census through the administration panel without having to modify the application code.
 
-It should be noted that to properly configure this connection will require a technical profile that knows the WebService of your City Council.
+It should be noted that to properly configure this connection, a technical profile that knows the WebService with which we want to connect will be required.
 
-Currently the application was designed to send only the **document number** and **document type**. With this new feature is enabled the possibility of sending if necessary the fields **date of birth** and **postal code**.
+## Enable feature
```

Comment:
I think an article is missing, and in English a gerund is probably better :thinking:.

```suggestion
## Enabling the feature
```